src/libraries/Microsoft.Extensions.FileProviders.Physical/src/Microsoft.Extensions.FileProviders.Physical.ref.cs

```diff
@@ -87,6 +85,6 @@ public partial class PollingWildCardChangeToken : Microsoft.Extensions.Primitive
         public bool ActiveChangeCallbacks { get { throw null; } }
         public bool HasChanged { get { throw null; } }
         protected virtual System.DateTime GetLastWriteUtc(string path) { throw null; }
-        System.IDisposable Microsoft.Extensions.Primitives.IChangeToken.RegisterChangeCallback(System.Action<object?> callback, object? state) { throw null; }
+        System.IDisposable Microsoft.Extensions.Primitives.IChangeToken.RegisterChangeCallback(System.Action<object> callback, object state) { throw null; }
```

Comment:
TODO: Understand this change

Reply:
@maxkoshevoi - is this another nullable mismatch between ref and src?

Reply:
This one is valid change

Reply:
If all tests call that method with state being null, wouldn't this change then be incorrect and the state should continue to be nullable?

Reply:
Oh, wait, by `This one is valid change` I meant that code in `main` branch is valid. 
Change in this PR is strange indeed. Both `ref` and `src` are annotated as `RegisterChangeCallback(System.Action<object?> callback, object? state)` on `main`